styles/components/_listingCard.scss

```diff
@@ -3,6 +3,9 @@
   height: 283px;
   font-size: $tx5b;
   cursor: pointer;
+  position: relative;
+  bottom: 0;
+  transition: all .2s ease;
```

Comment:
I set the transition here to **all** because the bottom param changes on hover. Is it possible to do this without setting **all**?